src/components/helpers/Overlay/style.scss

```diff
@@ -30,8 +30,5 @@
 
 .Overlay-open {
   overflow: hidden;
-  position: fixed;
-  top: 0;
-  bottom: 0;
```

Comment:
This class is only applied to the body. The body doesn't need to be position fixed, that's crazy stuff.